### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by poly000**

### CODE REVIEW

File: functional/generics-type-classes.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ This is called **monomorphization**, where different types are created from
...
 and different types can have different `impl` blocks.
 
-In object oriented languages, classes can inherit behavior from their parents.
+In object-oriented languages, classes can inherit behavior from their parents.
 However, this allows the attachment of not only additional behavior to
 particular members of a type class, but extra behavior as well.
 
 The nearest equivalent is the runtime polymorphism in Javascript and Python,
-where new members can be added to objects willy-nilly by any constructor.
+where new members can be added to object willy-nilly by any constructor.
```
Comment (simonsan) [inline functional/generics-type-classes.md:33]: "objects" in this case is right I guess, because it's plural

File: patterns/ffi/wrappers.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ This ensures that many patterns of access between types can be memory safe,
 data race safety included.
 
 However, when Rust types are exported to other languages, they are usually transformed
-into pointers. In Rust, a pointer means "the user manages the lifetime of the pointee."
+into pointers. In Rust, a pointer means "the user manages the lifetime of the pointer."
```
Comment (simonsan) [inline patterns/ffi/wrappers.md:13]: Pointee is right here.

https://en.wiktionary.org/wiki/pointee

**Review by simonsan — CHANGES_REQUESTED** [review]

**Comment by simonsan** [comment]
Hey, thanks for going through the files and correcting so many grammatic and spelling mistakes. I think those two instances up there don't need to be changed though. Can you please remove them and we are ready to merge. :)

**Comment by poly000** [comment]
> Hey, thanks for going through the files and correcting so many grammatic and spelling mistakes. I think those two instances up there don't need to be changed though. Can you please remove them and we are ready to merge. :)

ok

**Review by simonsan — APPROVED** [review]

**Comment by simonsan** [comment]
@poly000 Thank you!

**Pull request merged**